fhttp/http.go

```diff
@@ -1151,7 +1149,7 @@ func EchoHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-var payload = make([]byte, MaxPayloadSize)
+var payload = make([]byte, MaxPayloadSizeKb*1024)
```

Comment:
this is not going to work...
it's initialized during init() that probably need to be changed

you need a test that the change takes effect and that you don't get issues or non random data after the original size - you probably need a ChangePayloadSize() that translates from kb to bytes and sets the payload